crates/bevy_render/src/camera/projection.rs

```diff
@@ -112,3 +112,84 @@ impl Default for OrthographicProjection {
         }
     }
 }
+
+#[derive(Debug, Clone, Reflect, Serialize, Deserialize)]
+#[reflect_value(Serialize, Deserialize)]
+pub enum BaseAxis {
+    Vertical,
+    Horizontal,
+}
+
+#[derive(Debug, Clone, Reflect)]
+#[reflect(Component)]
+pub struct ScaledOrthographicProjection {
+    pub scale: f32,
+    pub aspect_ratio: f32,
+    pub near: f32,
+    pub far: f32,
+    pub window_origin: WindowOrigin,
+    pub base_axis: BaseAxis,
+}
+
+impl CameraProjection for ScaledOrthographicProjection {
+    fn get_projection_matrix(&self) -> Mat4 {
+        match (&self.window_origin, &self.base_axis) {
+            (WindowOrigin::Center, BaseAxis::Vertical) => Mat4::orthographic_rh(
+                -self.aspect_ratio * self.scale,
+                self.aspect_ratio * self.scale,
+                -self.scale,
+                self.scale,
+                self.near,
+                self.far,
+            ),
+            (WindowOrigin::BottomLeft, BaseAxis::Vertical) => Mat4::orthographic_rh(
```

Comment:
I think sorting by `WindowOrigin` makes more sense - it makes the parallels between horizontal and vertical more obvious.